slides/yocto-sdk/yocto-sdk.tex

```diff
@@ -273,7 +273,7 @@ \subsection{Devtool}
         \item \code{fetchuri} can be a local path or a remote {\em
           uri}.
       \end{itemize}
-    \item To modify an existing recipe: \code{devtool modify <recipe>}
+    \item To modify the source of an existing recipe: \code{devtool modify <recipe>}
```

Comment:
devtool modify allows to both modify the recipe itself and the sources of the targeted application so I'm not sure about that change